src/go/rdctl/pkg/paths/paths.go

```diff
@@ -33,6 +33,8 @@ type Paths struct {
 	Resources string `json:"resources"`
 	// Deployment Profile System-wide startup settings path.
 	DeploymentProfileSystem string `json:"deploymentProfileSystem,omitempty"`
+	// Secondary Deployment Profile System-wide startup settings path.
+	AltDeploymentProfileSystem string `json:"altDeploymentProfileSystem,omitempty"`
```

Comment:
I wonder if we should make `DeploymentProfileSystem` (and possibly some others) a list of paths instead… Maybe that's too much work right now?

Reply:
Yeah, not now. 😄